reverseGuess.js

```diff
@@ -0,0 +1,65 @@
+//Write the reverse game, where the computer thinks of a number and the human guesses it. 
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+} 
+
+let guessLog = [] // keep track of guess count and previous guesses
```

Comment:
Great! Remember back to our [increment and decrement](https://bootcamp.burlingtoncodeacademy.com/lessons/javascript/common-operators#anchor/incrementersdecrementers) operators. These might be useful in keeping track of how many guesses the user has made, or has left if you're giving them a finite number of chances.